docs/data/pages.ts

```diff
@@ -21,6 +20,13 @@ const pages: MuiPage[] = [
       { pathname: `/x/introduction/roadmap` },
     ],
   },
+  {
+    pathname: '/x/common-features-group',
```

Comment:
Do we have other pages to move here?
It felt weird to me to put this in "Introduction", which is why I chose to create a new folder

Reply:
Maybe the loacalization. Even if it's not yet supported for chart